src/store/common/types.ts

```diff
@@ -11,8 +11,8 @@ export interface ModeState {
     current: Mode;
 }
 
+export const SET_VISIBILITY = 'SET_VISIBILITY';
```

Comment:
I would just declare this inline with the action. We shouldn't need to pass the type string around.